both/methods.js

```diff
@@ -65,6 +65,11 @@ setSolveTime = function(resultId, solveIndex, solveTime) {
   RoundSorter.addRoundToSort(result.roundId);
 };
 
+function getTodayDateNoTime() {
+  var today = moment();
+  return new Date(Date.UTC(today.year(), today.month(), today.date()));
```

Comment:
To give the right user perceived day, this code needs to run in the client, right? I can't tell if it does.